jenkins/version-def.sh

```diff
@@ -48,9 +60,28 @@ SPARK_REPO=${SPARK_REPO:-"$URM_URL"}
 echo "CUDF_VER: $CUDF_VER, CUDA_CLASSIFIER: $CUDA_CLASSIFIER, PROJECT_VER: $PROJECT_VER \
     SPARK_VER: $SPARK_VER, SCALA_BINARY_VER: $SCALA_BINARY_VER"
 
+# Spark shim versions
+. $(dirname "$0")/common.sh
+case $PHASE_TYPE in
+    pre-release)
+        PROFILE_OPT="-Psnapshots" 
+        ;;
 
-SPARK_SHIM_VERSIONS_STR=${SPARK_SHIM_VERSIONS_STR:-"311 321cdh 312 313 314 320 321 322 330 331"}
-
-IFS=" " <<< $SPARK_SHIM_VERSIONS_STR read -r -a SPARK_SHIM_VERSIONS
-
+    *)
+        PROFILE_OPT="-PnoSnapshots" 
+        ;;
+esac
+# base version
+get_spark_shim_versions $PROFILE_OPT
```

Comment:
i find it hard to read that this is a get but we don't assign anything to it.  can we either have it return the values?

Reply:
if we can't return can we call it set_env_var_SPARK_SHIM_VERSIONS_to_shim_versions

Reply:
> if we can't return can we call it set_env_var_SPARK_SHIM_VERSIONS_to_shim_versions

this is a good name, since we cannot retrun an array or string from the function